tox.ini

```diff
@@ -8,9 +8,13 @@ envlist = py35, py36
 
 [testenv]
 changedir = /tmp
+setenv =
+    MPLCONFIGDIR={envtmpdir}/.matplotlib
+    PIP_USER = 0
+    PIP_ISOLATED = 1
```

Comment:
Not sure you really want to set PIP_ISOLATED?  This would also ignore whatever proxies are set in the config file (actually you are the one who raised that point :-))
The thing with PIP_USER=0 is that it's actually absolutely necessary to override --user to make tox work at all, whereas other settings may not need being overridden.

Reply:
Good call, dropping that bit.